### DISCUSSION TITLE
Can we re-arrange people by dragging?

### DISCUSSION THREAD

**Discussion by Bhagathxoxo** [Jisco/VisualFamilyTree#26] [Q&A] [answered]
I have added 3 sons to a parent and continue adding their partner and kids to the 3 sons. I have not arranged them(3 sons) by age, I can see an option to order them as birthdays but I don't know their birthdays, Just know the information that who is elder and who is younger.  so I want to rearrange the 3 sons without messing up the data or deleting the tree. Can we do that?

**Comment by Jisco** [comment, marked as answer]
Hi,
So no, this function is not included (yet).

Currently this would only be possible by changing the order in the stored data.
If you would like to do that, please contact me here again and I can explain to you how you can do it.

**Reply by Bhagathxoxo** [reply]
Yes please, I would like to know how to do that (I have Notepad++). Also I'm saving the file every time by going to Database - Export  and replacing the original file. Do I have to do this in order to save or just closing the file also saves everything.

**Reply by Jisco** [reply]
Great!
In the file you first have to find out the “identifier” of the respective person by e.g. search for the name under “persons”.

![image](https://github.com/Jisco/VisualFamilyTree/assets/770001/db8175a7-0369-4c2e-92e6-042f71a172e7)

With this you then go to “children” and look for the entry where “PersonId” = “Identifier” of the person you are looking for.

![image](https://github.com/Jisco/VisualFamilyTree/assets/770001/d798a05d-264a-4ad4-b4b6-82f450667c23)

The order in "children" determines the order in the display. So you simply move this entry either to the front or to the back, depending on which direction you want to make the change. 

Small note: The only important thing is to know that there should NOT be a "," at the end of the list. Otherwise the file cannot be read correctly.
![image](https://github.com/Jisco/VisualFamilyTree/assets/770001/e71a5719-7627-4f03-9261-287b514b05cb)

**Reply by Bhagathxoxo** [reply]
Thank you Its working.
